tests/DateTimeParseResultTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\DateTime\Tests;
```

Comment:
This should be in namespace `Brick\DateTime\Tests\Parser`, and located in a `tests/Parser` directory